src/components/views/Effect.svelte

```diff
@@ -104,6 +105,10 @@
       scaled: { min: limits.mix.min, max: limits.mix.max }
     })
 
+    // Check if the patch params have been modified from the original preset
+    const associatedPreset = $presetsStore.presets.find(({ id }) => id === patch.id)
+    unsavedChanges = (associatedPreset?.params.delayTime !== patch.params.delayTime) || (associatedPreset?.params.feedback !== patch.params.feedback) || (associatedPreset?.params.mix !== patch.params.mix)
```

Comment:
i kept this in here for now even though it isn't working because i figured it would be easier to test if it's already here 🤷🏼

Reply:
Yeah for sure! Worth keeping it around to debug. 💯